src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -72,6 +72,17 @@ private String getCloudInputs() {
         final String dbSNPb37_chr20 = localResources + DBSNP_138_B37_CH20_1M_1M1K_VCF;
         final String moreSites = getResourceDir() + "bqsr.fakeSitesForTesting.b37.chr17.vcf"; //for testing 2 input files (FIXME - this file is bogus because uses chr17)
 
+        final String hg18Reference = publicTestDir + "human_g1k_v37.chr17_1Mb.fasta";
+        final String HiSeqBam_chr17 = getResourceDir() + "NA12878.chr17_69k_70k.dictFix.bam";
+        final String dbSNPb37_chr17 =  getResourceDir() + "dbsnp_132.b37.excluding_sites_after_129.chr17_69k_70k.vcf";
+
+        final String GRCh37Ref_chr2021 = "src/test/resources/large/human_g1k_v37.20.21.fasta";
+        final String GRCh37Ref2bit_chr2021 = b37_2bit_reference_20_21;
+        final String more20Sites = localResources + "dbsnp_138.b37.20.10m-10m100.vcf"; //for testing 2 input files
+        final String more21Sites = localResources + "dbsnp_138.b37.21.10m-10m100.vcf"; //for testing 2 input files
```

Comment:
Have you confirmed that the variants in these vcfs actually overlap the reads in `hiSeqBam_20_21_100000`?